### PR TITLE
Add ability to configure locations in the default SS space

### PR DESCRIPTION
**Pull request by scollazo**

### CODE REVIEW

**Comment by scollazo** [comment]
This pr is built on top of #261

File: tasks/configure.yml [hunk 1]
```diff
@@ -194,3 +194,46 @@
...
+      register: space_uuid
+      tags: "configure-am"
+
+    - name: "Configure AM: get all TS descriptions from SS database"
+      become: "yes"
+      command:  mysql {{ archivematica_src_ss_db_name }} -Ns -e "select description from locations_location;"
+      register: location_descriptions
+      tags: "configure-am"
+
+    - name: "Configure SS: add NFS location: {{ item.location_description }}"
```
Comment (mamedin) [inline tasks/configure.yml]: Hi Santi, the PR looks fine, but I think we need a cosmetic change. This task name should be more generic, something like "Configure SS: add custom locations". The `{{ item.location_description }}` is not changed to its value on every iteration, so we can remove it from name:

```
TASK [external-roles/artefactual-archivematica-src-mysql : Configure SS: add NFS location: {{ item.location_description }}] ************************************************************************
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/replicator', u'location_purpose': u'RP', u'location_description': u'Replicator location', u'location_default': u'false'})
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/aipstore', u'location_purpose': u'AS', u'location_description': u'AipStore', u'location_default': u'true'})
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/transfer-source/', u'location_purpose': u'TS', u'location_description': u'Transfer Source', u'location_default': u'false'})
```

**Review by mamedin — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 1 above)
Comment (mamedin) [inline tasks/configure.yml]:
```suggestion
    - name: "Configure SS: add custom locations"
```

**Review by mamedin — APPROVED** [review]
LGTM

**Pull request merged**